### PR TITLE
Added Title Metadata to TARDIS Website

### PR DESCRIPTION
**Pull request by DhruvSondhi**
Updated the Meta Data to TARDIS Website to point to the correct values.

**Description**

**Motivation and context**

**How has this been tested?**
- [x] Testing pipeline.
- [ ] Other. 

**Type of change**

- [ ] Bug fix. 
- [x] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [ ] My change requires a change to the documentation.
    - [ ] I have updated the documentation accordingly.
    - [ ] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [ ] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

**Comment by DhruvSondhi** [comment]
**Preview here**: https://dhruvsondhi.github.io/tardis-sn.github.io/branch/title_metadata/

File: content/_index.md [hunk 1]
```diff
@@ -103,17 +103,17 @@ sections:
     background: gray
     title: Latest Posts
 seo:
-  title: Stackbit Azimuth Theme
-  description: The preview of the Azimuth theme
+  title: TARDIS-SN
+  description: Website for TARDIS-SN
```
Comment (jaladh-singhal) [inline content/_index.md:107]: This is not how you describe something. Suppose you shared Numfocus website on twitter, it shouldn't say Website for NumFOCUS, right? :sweat_smile:

It instead describes what NumFOCUS is. Do the same for TARDIS for all 3 descriptions - you can use our twitter bio (https://twitter.com/tardis_sn)

**Review by jaladh-singhal — CHANGES_REQUESTED** [review]

File: content/_index.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ sections:
     title: Latest Posts
 seo:
   title: TARDIS-SN
-  description: Website for TARDIS-SN 
+  description: TARDIS is an open-source code to analyze observations of exploding stars. It is developed by a multi-disciplinary team.
```
Comment (jaladh-singhal) [inline content/_index.md:107]:
```suggestion
  description: A community-developed tool to simulate the spectrum of light emitted by exploding stars using Monte Carlo techniques.
```

File: content/_index.md [hunk 3]
```diff
@@ -113,7 +113,7 @@ seo:
       value: TARDIS-SN
       keyName: property
     - name: 'og:description'
-      value: Website for TARDIS-SN
+      value: TARDIS is an open-source code to analyze observations of exploding stars. It is developed by a multi-disciplinary team.
```
Comment (jaladh-singhal) [inline content/_index.md:116]:
```suggestion
      value: A community-developed tool to simulate the spectrum of light emitted by exploding stars using Monte Carlo techniques.
```

File: content/_index.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ seo:
     - name: 'twitter:title'
       value: TARDIS-SN
     - name: 'twitter:description'
-      value: Website for TARDIS-SN
+      value: TARDIS is an open-source code to analyze observations of exploding stars. It is developed by a multi-disciplinary team.
```
Comment (jaladh-singhal) [inline content/_index.md:127]:
```suggestion
      value: A community-developed tool to simulate the spectrum of light emitted by exploding stars using Monte Carlo techniques.
```

**Review by jaladh-singhal — APPROVED** [review]

**Pull request merged**